ui/apps/platform/src/Containers/Vulnerabilities/VulnerablityReporting/CreateVulnReport/CreateVulnReportPage.tsx

```diff
@@ -37,8 +38,21 @@ function VulnReportsPage() {
                     </FlexItem>
                 </Flex>
             </PageSection>
-            <PageSection padding={{ default: 'noPadding' }}>
-                <PageSection isCenterAligned />
+            <Divider component="div" />
+            <PageSection padding={{ default: 'noPadding' }} isCenterAligned>
+                <Wizard
+                    navAriaLabel="Report creation steps"
+                    mainAriaLabel="Report creation content"
```

Comment:
I wonder if this should change, based on the step?

Check with @alwayshooin and @pedrottimark about what they found in Policy wizard, or if we also need to update Policy wizard.

Reply:
Looks like they did change it, but it was based on whether the user was editing, creating, or cloning a policy. I could do that as well